src/OAuth2/ResponseType/JwtAccessToken.php

```diff
@@ -127,5 +127,20 @@ protected function createPayload($client_id, $user_id, $scope = null)
             'token_type' => $this->config['token_type'],
             'scope'      => $scope
         );
+        
+        if (isset($this->config['jwt_extra_payload'])) {
+            if (is_callable($this->config['jwt_extra_payload'])) {
+                $extra = call_user_func($this->config['jwt_extra_payload'], $client_id, $user_id, $scope);
+                if (is_array($extra)) {
```

Comment:
@bshaffer this is about array_merge error vs custom error? result will be the same :D but maybe yes more clear error message is better